spec/system/reports/admin_sees_reports_listing_page_spec.rb

```diff
@@ -0,0 +1,100 @@
+
+    expect(page).to have_content report.truncated_message
+    expect(page).to have_content 'Discurso de ódio'
+    expect(page).to have_content 'Publicação'
+    expect(page).to have_content other_report.truncated_message
+    expect(page).to have_content 'Abuso/Perseguição'
+    expect(page).to have_content 'Comentário'
+    expect(page).not_to have_content another_report.truncated_message
+    expect(page).not_to have_content 'Spam'
+    expect(page).not_to have_content 'Perfil'
+    expect(page).to have_current_path reports_path
+  end
+
+  it 'e visualiza denúncias deferidas' do
+    admin = create(:user, :admin)
+    report = create(:report, :for_post, offence_type: 'Discurso de ódio')
+    other_report = create(:report, :for_comment, offence_type: 'Abuso/Perseguição', status: :granted)
+    another_report = create(:report, :for_profile, offence_type: 'Spam')
+
+    login_as admin
+    visit root_path
+    click_button class: 'dropdown-toggle'
+    click_on 'Denúncias'
+    click_on 'Deferido'
+
+    expect(page).not_to have_content report.truncated_message
+    expect(page).not_to have_content 'Discurso de ódio'
+    expect(page).not_to have_content 'Publicação'
+    expect(page).to have_content other_report.truncated_message
+    expect(page).to have_content 'Abuso/Perseguição'
+    expect(page).to have_content 'Comentário'
+    expect(page).not_to have_content another_report.truncated_message
+    expect(page).not_to have_content 'Spam'
+    expect(page).not_to have_content 'Perfil'
+    expect(page).to have_current_path reports_path({ filter: 'granted' })
+  end
+
+  it 'e visualiza denúncias indeferidas' do
+    admin = create(:user, :admin)
+    report = create(:report, :for_post, offence_type: 'Discurso de ódio', status: :not_granted)
+    other_report = create(:report, :for_comment, offence_type: 'Abuso/Perseguição')
+    another_report = create(:report, :for_profile, offence_type: 'Spam')
+
+    login_as admin
+    visit root_path
+    click_button class: 'dropdown-toggle'
+    click_on 'Denúncias'
+    click_on 'Indeferido'
+
+    expect(page).to have_content report.truncated_message
+    expect(page).to have_content 'Discurso de ódio'
+    expect(page).to have_content 'Publicação'
+    expect(page).not_to have_content other_report.truncated_message
+    expect(page).not_to have_content 'Abuso/Perseguição'
+    expect(page).not_to have_content 'Comentário'
+    expect(page).not_to have_content another_report.truncated_message
+    expect(page).not_to have_content 'Spam'
+    expect(page).not_to have_content 'Perfil'
+    expect(page).to have_current_path reports_path({ filter: 'not_granted' })
+  end
+
+  it 'e não há denúncias disponíveis' do
+    admin = create(:user, :admin)
+
+    login_as admin
+    visit reports_path
+
+    expect(page).to have_content 'Nenhuma denúncia encontrada'
+  end
+
+  it 'e precisa estar logado' do
+    visit reports_path
+
+    expect(current_path).to eq new_user_session_path
```

Comment:
em algumas máquinas aqui do pessoal, de vez em quando acontece de esse tipo de teste falhar, pois, às vezes, não está dando tempo de mudar de url no teste. Para contornar isso, estamos utilizando o seguinte método.
```suggestion
    expect(page).to have_current_path new_user_session_path
```